src/sentry/data_export/endpoints/data_export.py

```diff
@@ -29,6 +29,8 @@
 SUPPORTED_DATASETS = {
     "discover": Dataset.Discover,
     "issuePlatform": Dataset.IssuePlatform,
+    "transactions": Dataset.Transactions,
+    "errors": Dataset.Events,
```

Comment:
Works for transactions, but I think for errors, we should use the ErrorsQueryBuilder? It inherits DiscoverQueryBuilder but has an ErrorsQueryBuilderMixin 👀

Reply:
:+1: added a condition for the ErrorsQueryBuilder here
https://github.com/getsentry/sentry/pull/75853/commits/69678d3de6f0ad880fad8bb7eb03cd3b535acba8